package.json

```diff
@@ -80,11 +68,12 @@
   },
   "lint-staged": {
     "*.{js,vue}": [
-      "eslint --fix",
+      "kcd-scripts format",
```

Comment:
You can get rid of the lint-staged config entirely and add husky config instead like my other projects and that do the same thing

Reply:
I tried to, but kcd-scripts' `.lintstagedrc.js` ([source](https://github.com/kentcdodds/kcd-scripts/blob/master/src/config/lintstagedrc.js#L8)) wouldn't get trigged by `.vue` files 😢 

Would you accept a PR to add them? That wouldn't make kcd-scripts to work with Vue files out-of-the-box, but wouldn't hurt anybody either.

ps: thanks for the feedback! 💯

Reply:
Sure 👍